### PR TITLE
feat: #80720 - SideSheet

### PR DESCRIPTION
**Pull request by Tobzor**
## Prerequisites

Please make sure you can check the following boxes:

-   [x] My code follows the code style of this project
-   [x] All new and existing tests passed
-   [x] Fix any eslint/prettier warnings/errors: npm run lint
-   [x] Remove unnecessary console.log (use logInfo logWarn if it's needed)
-   [x] Are variable and function names properly describing what it does?

### Contribution Type

What types of changes does your code introduce? Put an `x` in all the boxes that apply:

-   [ ] Bug fix (non-breaking change which fixes an issue, please reference the issue id)
-   [x] New feature (non-breaking change which adds functionality, make sure to open an associated issue first)
-   [ ] Breaking change (fix or feature that would cause existing functionality to change)
-   [ ] I have updated the documentation accordingly
-   [ ] I have added tests to cover my changes

### Description

[User story 80720](https://dev.azure.com/EquinorASA/DT%20%E2%80%93%20Digital%20Twin/_boards/board/t/Echo%20Library/Stories/?workitem=80720)

[Figma specs](https://www.figma.com/file/TjT41TmEyM6E1eFdRnFYRp/Echo-Components?node-id=950%3A22708)

### Remarks

[Optionally place any follow-up comments, remarks, observations, or notes here for future reference]

### CODE REVIEW

File: src/components/sidesheet/sheet/Sheet.module.css [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    box-shadow: 3px 0px 4px 0px rgba(0, 0, 0, 0.12), 2px 0px 4px 0px rgba(0, 0, 0, 0.14);
+
+    
+    position: absolute;
+    left: 0;
+    right: 0;
+}
+
+.fullscreenContainer {
+    composes: container;
```
Comment (khollund) [inline src/components/sidesheet/sheet/Sheet.module.css:48]: I'm getting unknown property warning on these composes, possible to fix with some config change?

**Review by khollund — APPROVED** [review]
Looks good

File: src/components/sidesheet/sheet/Sheet.module.css (hunk 1 above)
Comment (Tobzor) [reply src/components/sidesheet/sheet/Sheet.module.css:48]: I'm not actually sure. Its in the spec for css modules, but i'll look into it

File: .eslintrc.js [hunk 2]
```diff
@@ -16,15 +16,26 @@ module.exports = {
...
+        'prettier/prettier': [
+            'error',
+            {
+                //see: https://prettier.io/docs/en/options.html
+                semi: true,
+                trailingComma: 'none',
+                singleQuote: true,
+                printWidth: 120,
+                tabWidth: 4,
+                endOfLine: 'auto' //"lf|crlf|cr|auto"
```
Comment (OveBDev) [inline .eslintrc.js:37]: does this make line 25 obsolete? ( 'linebreak-style': ['warn', process.platform === 'win32' ? 'windows' : 'unix'], // https://stackoverflow.com/q/39114446/2771889)

Comment (Tobzor) [reply .eslintrc.js:37]: I'm not actually sure. We have never had the linebreak issues in our modules -- but that might be due to _not_ having it as a eslint warning rule. Not sure linebreak rule is even needed?

Comment (OveBDev) [reply .eslintrc.js:37]: Just tested it here on unix/Mac, and line ending works fine, so all is good :)

**Pull request merged**